ark-plonk-book/src/chapter_1.md

```diff
@@ -0,0 +1,373 @@
+2. In the projective form, each elliptic curve point has 3 coordinates instead of 2 $(x,y,z)$ for $z\neq 0$ (for all points except the point at infinity). Using the projective form allows us to give coordinates to the point at infinity. It also speeds up some of the most used arithmetic operations in the curve. The forward mapping is given by $(x,y)\rightarrow (xz,yz,z)$ and reverse mapping is giving by $(x,y,z)(\dfrac{x}{z},\dfrac{y}{z})$. Let $P=(x_p,y_p,z_p)$ and $Q=(x_q,y_q,z_q)$ and $\dfrac{x_p}{z_q}\neq\dfrac{x_q}{z_p}$. 
+
+
+<figure>
+<img src="images/images/ECP.png"
+     alt="projective"
+     style="float: left; margin-right: 10px;" />
+<figcaption>Figure 2: Graphical representation of the map with z =1</figcaption>
+</figure>
+
+By expanding the previous arithmetics, we have:
+
+$\lambda=\dfrac{\dfrac{y_q}{z_q}-\dfrac{y_p}{z_p}}{\dfrac{x_q}{z_q}-\dfrac{x_p}{z_p}}=\dfrac{\dfrac{y_q}{z_q}-\dfrac{y_p}{z_p}}{\dfrac{x_q}{z_q}-\dfrac{x_p}{z_p}}*\dfrac{z_pz_q}{z_pz_q}$ 
+
+$\lambda=\dfrac{y_qz_p-y_pz_q}{x_qz_p-x_pz_q}$ 
+
+
+ $x_s=\lambda^2-\dfrac{x_p}{z_p}-\dfrac{x_q}{z_q}$ and     $y_s=\lambda(\dfrac{x_p}{z_p}-x_s)-\dfrac{y_p}{z_p}$
+
+#### Point doubling
+
+We will compute $P+P=2P$ in the affine form as follows:\
+If $P=O$ then $2P=O$ \
+Else  $P=(x,y)$:\
+    &emsp;&nbsp;If $y=0$ then $2P=O$  
+    &emsp;&nbsp;Else $2P=(x',y')$  
+
+such that: the derivative $\lambda=\dfrac{dy}{dx}=\dfrac{3x^2}{2y},\; x'=\lambda^2-2x\;$   and $y'=\lambda(x-x')-y=\lambda^3+3\lambda x-y$
+
+#### The discrete logarithm problem
+The security of many cryptographic techniques depends on the intractability of the discrete logarithm problem.
+
+#### Definition:
+Let $G$ be a multiplicative group. The discrete logarithm problem (DLP) is:
+Given $g,h\in G$ to find $a$, if it exists, such that $h=g^a$.
+
+
+#### Elliptic Curve Discrete Logarithm Problem  (ECDLP)
+
+Let $E$ be an elliptic curve of the Weierstrass form defined over a finite field $F_q$. Let $S$ and $T$ be two points in $E(F_q)$. Find an integer $m$ such that:
+$$T=mS$$
+
+The fastest method to solve the ECDLP problem in $E(F_q)$ is the [Pollard Rho](https://www.ams.org/mcom/1978-32-143/S0025-5718-1978-0491431-9/S0025-5718-1978-0491431-9.pdf) method which has exponential complexity $O(\sqrt{|G|})$.
+
+### Pairings
+Pairing based-cryptography is used in many cryptographic applications like signature schemes, key agreement, zero knowledge...etc.\
+For example, pairings are used to create efficient circuit-based zero knowledge proofs.
+
+#### Definition
+A pairing $e$ is a bilinear map $〈·,·〉$  defined as: 
+$$e:G_1\times G_2\rightarrow G_T$$
+Such that $G_1,G_2$ and $G_T$ are abelian groups. The bilinear property means that:
+$e(P+P',Q)=e(P,Q)+e(P',Q)$\
+$e(P,Q+Q')=e(P,Q)+e(P,Q')$
+
+​​From which, we deduce the following transformations:
+
+
+$$e([a]P,[b]Q)=e(P,[b]Q)^a=e([a]P,Q)^b=e(P,Q)^{ab}=e([b]P,[a]Q)$$
+
+for $P,P'\in G_1$ and $Q,Q'\in G_2$ and $a,b\in \Z$
```

Comment:
Here we are using additive form for `G_T` in the definition of bilinearity and then multiplicative form in the scalar multiplication case. Probably want to use multiplicative form in both cases.

Reply:
I agree. In particular I prefer using additive notation for EC groups (G1 and G2) and multiplicative addition for multiplicative subgroups of fields (GT)

Reply:
Yeah will change it since it's confusing. Making it as @davidnevadoc suggested